build/Build.cs

```diff
@@ -115,63 +115,63 @@ class Build : NukeBuild
     Target Merge => _ => _
         .DependsOn(Compile)
         .Executes(() =>
+    {
+        foreach (var target in new[] { "net462", "net48", "netstandard2.0" })
```

Comment:
This whole target has not changed - It's just less indented to match other formatting... I can revert it if it's annoying.